tools/autograd/gen_autograd_functions.py

```diff
@@ -108,37 +108,63 @@
 # Getter templates
 GETTER_DEFINITION = CodeTemplate("""\
 PyObject* THP${op}_${name}_getter(THPCppFunction *self, void *_unused) {
+  HANDLE_TH_ERRORS
   auto prop = static_cast<${op}*>(self->cdata.get())->${name};
   ${body}
+  END_HANDLE_TH_ERRORS
 }
 """)
 
 GETTER_DEFINITION_SAVEDVAR = CodeTemplate("""\
 PyObject* THP${op}_${name}_getter(THPCppFunction *self, void *_unused) {
+  HANDLE_TH_ERRORS
   const auto& prop = static_cast<${op}*>(self->cdata.get())->${name}_;
   ${body}
+  END_HANDLE_TH_ERRORS
+}
+""")
+
+GETTER_DEFINITION_VEC_SAVEDVAR = CodeTemplate("""\
+PyObject* THP${op}_${name}_getter(THPCppFunction *self, void *_unused) {
+  HANDLE_TH_ERRORS
+  const auto *node = static_cast<${op}*>(self->cdata.get());
+  const auto& prop = node->${name}_;
+  if (node->${name}_released_) {
```

Comment:
Ho so we already use a boolean for tensorlist haha

Reply:
Yup

Reply:
Good thing we have this because otherwise we actually wouldn't have an error for the vector of saved variables case, by default it would've just returned an empty tuple